src/main/java/com/redhat/ecosystemappeng/crda/integration/report/ReportIntegration.java

```diff
@@ -61,7 +61,9 @@ public void configure() {
             .setBody(exchangeProperty(Constants.REPORT_PROPERTY))
             .bean(ReportTransformer.class, "filterVerboseResult")
             .marshal().json()
-            .marshal().mimeMultipart(Constants.MULTIPART_MIXED_TYPE.getSubtype(), false, false, true);
+            .marshal().mimeMultipart(Constants.MULTIPART_MIXED_TYPE.getSubtype(), false, false, true)
+            // original Content-Type header contains special characters that breaks java's built-in http client
+            .setHeader("Content-Type", header("Content-Type").regexReplaceAll("[\t|\r|\n]", ""));
```

Comment:
Regardless of the resolution of this PR this should be changed.
```suggestion
            .setHeader(Exchange.CONTENT_TYPE, header(Exchange.CONTENT_TYPE).regexReplaceAll("[\t|\r|\n]", ""));
```